src/Files.App/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -402,12 +403,17 @@
 						<TextBox
 							x:Name="TileViewTextBoxItemName"
 							Grid.Row="0"
+							Padding="5 2 5 2"							
```

Comment:
What's the padding for?

Reply:
It looks a bit better if the text isn't directly next to the edge.

Reply:
Let's continue without the padding and discuss in a seperate issue.